_posts/2020-05-24-Journey-to-dev.md

```diff
@@ -52,6 +52,45 @@ company private for now, but wish me good luck and fingers crossed.
 It is quiet riveting, applying for jobs. It took a lot of courage and
 commitment to press that send button. I feel a lot better now.
 
+#### Newly found interest and possible project for the weekend
+A few ideas have popped up today, and I have gathered some momentum 
+yet again that could take me off my path. I have been trying to learn and
+understand how a framework like Django works, then I started trying to write
+my own framework while thinking of enrolling int he LeeCode June Challenge of
+30 days with 30 problems solved. Even as I write this I realise it is a bit much,
+when I already started the Kaggle Challenge, and am looking for jobs while 
+still having a full time job. **SO**, I will try to Keep It Simple and reduce. To be 
+able to do this, I've decided to create a queue, which will work following these rules:
+*(probably a bit like Kanban)*
+
+1. I am not able to start a new project until one is finished from the **working on** queue
+2. No more than **2 working on** projects
```

Comment:
👍 
if you like reading about this kind of stuff, I recommend [The Phonix Project](https://www.amazon.co.uk/Phoenix-Project-DevOps-Helping-Business-ebook/dp/B00AZRBLHO), aside from the lots of references to limiting WIP, it's a very fun read.

Reply:
Hi Radu,

Thanks for the tip, much appreciated! Keeping it lean and clean is the struggle with learning something